helm/prefect-server/templates/apollo/service.yaml

```diff
@@ -14,6 +14,9 @@ spec:
       targetPort: http
       protocol: TCP
       name: http
+  {{- with .Values.apollo.service.publicIP }}
+  loadBalancerIP: {{ . }}
+  {{- end }}
```

Comment:
I think this should just allow extra data to be dumped into this since the type is not always a `LoadBalancer`

If you'd like to contribute this, you'll need to add `apollo.service.extras`  to `values.yaml` and then unpack `extras` here. You can document this `loadBalancerIP` usage in the helm chart readme.